src/router/index.ts

```diff
@@ -20,6 +22,20 @@ const routes: RouteRecordRaw[] = [
         path: '/dapps',
         name: 'dApps',
         component: DApps,
+        children: [
+            {
+                path: '',
+                redirect: '/dapps/dapps-staking',
```

Comment:
The path string check in the `Sidebar.vue` (from https://github.com/PlasmNetwork/plasm-apps-vue/blob/main/src/components/Sidebar.vue#L32-L38) component cannot properly check the redirected path, so when the user clicks on the "dApps" menu from the sidebar, it does not get highlighted.

You can see from the screenshot that the "Balance" tab is highlighted when selected, but the "dApps" tab is not highlighted even though it was selected.

<img width="1119" alt="Screen Shot 2021-03-10 at 8 37 11 PM" src="https://user-images.githubusercontent.com/40356749/110623746-a53fa080-81e0-11eb-90e7-315bd6e3cc77.png">

<img width="1119" alt="Screen Shot 2021-03-10 at 8 37 01 PM" src="https://user-images.githubusercontent.com/40356749/110623763-a96bbe00-81e0-11eb-9cba-5baefd74f5d9.png">

Reply:
I changed sidebar to read first param of path split by the letter '/'
/dapps/dapps-staking -> dapps